RealmBrowserSync/Controllers/RLMSyncServerBrowserWindowController.m

```diff
@@ -23,6 +23,7 @@
 
 static NSString * const RLMAdminRealmServerPath = @"__admin";
 static NSString * const RLMAdminRealmRealmFileClassName = @"RealmFile";
+static NSString * const RLMAdminRealmRealmFilesDefaultPredicate = @"(path != '/__admin') AND NOT (path ENDSWITH '__permission')";
```

Comment:
Hah, in the dashboard, I implemented a filter for `__permission` and `__management` -- but not for `__admin`. I guess we should filter all three both places?